### PR TITLE
Minor refactor according to review suggestions

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: src/ds3/buildclient/build_client.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func createClient(endpoint, accessKey, secretKey, proxy string) (*ds3.Client, er
 
     // Parse proxy.
     var proxyUrlPtr *url.URL
-    if proxy != "" {
+    if len(proxy) == 0 {
```
Comment (DenverM80) [inline src/ds3/buildclient/build_client.go]: does this check for NULL as well?

Comment (RachelTucker) [inline src/ds3/buildclient/build_client.go]: len(nil) = 0

Comment (DenverM80) [inline src/ds3/buildclient/build_client.go]: 2nd question - shouldn't it be:
if len(proxy) != 0 { 
...

Comment (shabtaisharon) [inline src/ds3/buildclient/build_client.go]: yes, i think you are right

Comment (RachelTucker) [inline src/ds3/buildclient/build_client.go]: Good catch. Updated.

**Pull request merged**